src/app/static/src/app/components/ResetConfirmation.vue

```diff
@@ -87,6 +89,13 @@
                     this.newVersion();
                 }
             },
+            changesToShow: function (){
```

Comment:
The purpose of this function is to remove the calibrate model step from the list of subsequent steps with changes when edits require confirmation. When the user attempts to rerun the model, it will show the model run step as a step with changes that would be lost.